infrastructure/environments/cloudformation/full/common/scaling.yaml

```diff
@@ -0,0 +1,113 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: ECS Service with Auto-Scaling
+
+
+Parameters:
+  DaemonServiceName:
+    Type: String
+    Description: Name for the Dagster Daemon Service
+  DaemonServiceARN:
+    Type: String
+    Description: ARN for the Dagster Daemon Service
+  DagitServiceName:
+    Type: String
+    Description: Name for the Dagit Service
+  DagitServiceARN:
+    Type: String
+    Description: ARN for the Dagit Service
+  CodeServerServiceName:
+    Type: String
+    Description: Name for the Dagit Service
+  CodeServerServiceARN:
+    Type: String
+    Description: ARN for the Dagit Service
+  ECSClusterName:
+    Type: String
+    Description: Name for the ECS Dagster Service
+  TimeOn:
+    Type: String
+    Description: Time to turn the pipeline on (e.g., 23 for 11pm)
+    Default: 23
+  TimeOff:
+    Type: String
+    Description: Time to turn the pipeline off (e.g., 23 for 11pm, 5 for 5am)
+    Default: 5
```

Comment:
So the pipelines run only at night?

Reply:
For production systems, that's the idea, yes.  The idea is that they will have the result of the pipeline when they start in the morning